test/ODESolvers/runtests.jl

```diff
@@ -386,21 +386,19 @@ let
     @inbounds begin
       increment || (dQ .= 0)
       yf = Q[1]
-      ys = Q[2]
-      gf = (-3 + yf^2 - cos(ω * t)) / 2yf
-      gs = (-2 + ys^2 - cos(    t)) / 2ys
-      dQ[1] += Ω[1,1] * gf + Ω[1, 2] * gs - ω * sin(ω * t) / 2yf
+      g = @SVector [(-3 + yf^2 - cos(ω * t)) / 2yf, 0]
+      dQ .+= Ω * g
+      dQ[1] -= ω * sin(ω * t) / 2yf
```

Comment:
I guess I am missing something, but I think what you previously had was correct.